app/controllers/posts_controller.rb

```diff
@@ -4,7 +4,6 @@ class PostsController < ApplicationController
 
   def index
     @user = User.find_by(id: params[:user_id])
-    @all_posts = @user.posts
   end
```

Comment:
- [ ] <b>REQUIRED:</b> why did you delete `all_posts` here, Because we will need to get all posts at some point.  and also, you will have to avoid N+1 Problem by doing something like this `@all_posts = @user.posts.includes(:comments)`.

Reply:
✅ Resolved.